rpa-mapper/src/test/java/uk/gov/hmcts/cmc/rpa/mapper/DefenceResponseJsonMapperTest.java

```diff
@@ -0,0 +1,162 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.json.JSONException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.response.DefenceType;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleAddress;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleParty;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleTheirDetails;
+import uk.gov.hmcts.cmc.domain.utils.ResourceReader;
+import uk.gov.hmcts.cmc.rpa.config.ModuleConfiguration;
+
+import java.time.LocalDate;
+
+import static org.skyscreamer.jsonassert.JSONAssert.assertEquals;
+import static org.skyscreamer.jsonassert.JSONCompareMode.STRICT;
+
+@SpringBootTest
+@ContextConfiguration(classes = ModuleConfiguration.class)
+@RunWith(SpringJUnit4ClassRunner.class)
+@SuppressWarnings({"LineLength"})
+public class DefenceResponseJsonMapperTest {
+
+    private static String INDIVIDUAL = "/DefenceResponse/defence_response_individual_rpa_case.json";
```

Comment:
please don't use capitals in directories, we don't do it for anything else
also no need to call it a defence and a response, its a response
also no need to repeat it in the directory and file name

perhaps:
`rpa/response/individual.json`

REQ

Reply:
renamed to lower case and single value.